dashboard/config/scripts/2-3/artist/2-3_artist_loops_match_1.match

```diff
@@ -28,4 +28,4 @@ question '/script_assets/2-3_images/artist/2-3_artistloops_match_1_q4.png'
 #question 'Rhombus'
 #answer 'image of code for rhombus'
 
-# http://staging.learn.code.org/s/course2/stage/5/puzzle/12
\ No newline at end of file
+# http://staging.learn.code.org/s/course2/lessons/5/levels/12
```

Comment:
I've never seen this URL before. It doesn't seem to work on either the new or old pattern though

Reply:
I have never seen that either.  I guess if it didn't work before this isn't hurting anything